go/libkb/identify_state.go

```diff
@@ -181,7 +181,8 @@ func (s *IdentifyState) computeKeyDiffs(dhook func(keybase1.IdentifyKey) error)
 	observedEldest := s.u.GetEldestKID()
 	if s.track != nil {
 		trackedEldest := s.track.GetEldestKID()
-		if observedEldest.NotEqual(trackedEldest) {
+		if observedEldest.NotEqual(trackedEldest) ||
+			s.u.GetCurrentEldestSeqno() > s.track.GetTrackedLinkSeqno() {
```

Comment:
@patrickxb quick look at a change in how resets in tracked users are detected?

Reply:
The assumption is that if someone's eldest_seqno is higher than the seqno of tail link we tracked (embedded in our "track" link), it means they must have reset.

Reply:
Maybe things have changed, but I thought when a user resets the eldest_seqno goes to 0.  So wouldn't the tracked seqno be higher than the current user seqno?

Reply:
`GetCurrentEldestSeqno` returns the seqno that starts the current subchain, so there's no way to figure out if someone was reset but hasn't provisioned yet. That being said for all intents and purposes the account is useless until you provision _something_ and as soon as you're done with that the change will be noticed by this code.

Reply:
please hold for a bit, i'll take a look as well. seems to be possible to detect that we are loading in-between-reset user, but i don't know off the bat

Reply:
I really don't understand the significance of detecting whether account is in a temporary unusable state, as far as I understand the whole model it's not a vulnerability at all.

Reply:
I tried this today. When an account is in eldest=0 state, observedEldest is "", so user gets something like 
```
✖ <Account reset! Old key was 0120bd9a998f63961caeffe5e033f413e46bb90e855ae13a96323feb6dbfa1a71cca0a; new key is >
```
in this case.

This is good to go IMO